solutions/18.py

```diff
@@ -0,0 +1,20 @@
+#Himanshu Nachane (@Himanshu1495)
+'''return missing number in an input array of consecutively increasing numbers.
+
+i: [5,6,8,9]
+o: 7
+'''
+
+def missing_num(arr):
+	#get first element of array
+	start = arr[0]
+	#get last element of array
+	end = arr[-1]
+	#create new array by iterating from start element to end element
+	new_arr = [x for x in range(start,end+1)]

```

Comment:
@Himanshu1495 , try this, it looks much cleaner and doesn't involve creating a new array.
```python
sum = 0;
for x in range(start, end+1)
  sum += x; 
```